.github/workflows/test-coverage.yaml

```diff
@@ -18,18 +22,24 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Setup python ${{ matrix.python-version }}
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+        with:
+          enable-cache: true
+          cache-dependency-glob: pyproject.toml
+          cache-suffix: test
+
+      - name: Setup Python ${{ matrix.python-version }}
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-          cache: "pip"
 
       - name: Install dependencies
-        run: pip install .[test]
+        run: uv pip install .[test]
 
       - name: Run pytest
         run: |
-          pytest --doctest-modules \
+          uv run pytest --doctest-modules \
             --junitxml=junit/test-results-${{ matrix.python-version }}.xml
```

Comment:
**[Question, non-blocking]** Same question here about using the `uv` prefix when `UV_SYSTEM_PYTHON=1`.

Reply:
Fixed in acafd68!